src/microbe_stage/MicrobeAI.cs

```diff
@@ -740,7 +740,10 @@ private void LaunchToxin(Vector3 target)
             if (CanShootToxin())
             {
                 microbe.LookAtPoint = target;
-                microbe.QueueEmitToxin(oxytoxy);
+
+                // hold fire until the target is lined up.
+                if (microbe.FacingDirection().Normalized().AngleTo(microbe.LookAtPoint.Normalized()) < 0.1f + SpeciesActivity / (5 * SpeciesFocus))
+                    microbe.QueueEmitToxin(oxytoxy);
```

Comment:
```suggestion
                // Hold fire until the target is lined up.
                if (microbe.FacingDirection().Normalized().AngleTo(microbe.LookAtPoint.Normalized()) < 
                    Constants.MICROBE_AI_TOXIN_FIRE_ANGLE_THRESHOLD + SpeciesActivity / (5 * SpeciesFocus))
                {
                    microbe.QueueEmitToxin(oxytoxy);
                }
```

A bit better formatting that should also please the linter (hopefully, I wrote this on Github), and also I suggest adding a tweak constant.